torch/quantization/fx/utils.py

```diff
@@ -1,7 +1,76 @@
+import re
+
 # turn foo.bar -> ['foo', 'bar']
 def _parent_name(target):
     r = target.rsplit('.', 1)
     if len(r) == 1:
         return '', r[0]
     else:
         return r[0], r[1]
+
+
+def graph_pretty_str(g, shorten=True) -> str:
```

Comment:
btw, james has an example to print a table in his slides:

```
class MyModule(torch.nn.Module):
    def __init__(self):
        super().__init__()
        self.param = torch.nn.Parameter(torch.rand(3, 4))
        self.linear = torch.nn.Linear(4, 5)
        
    def forward(self, x):
        return torch.relu(self.linear(x + self.param))
    
m = MyModule()
gm = GraphModule(m)

from tabulate import tabulate
node_specs = [[n.op, n.target, n.args, n.kwargs] for n in gm.graph.nodes]
print(tabulate(node_specs, headers=['opcode', 'target', 'args', 'kwargs']))
```
would be good if we can reuse the code that generate a table

Reply:
agreed.  One issue is that `tabulate` is not in Python STL and isn't currently required by pytorch.

Reply:
yeah sounds good. we can change this if torch.fx decide to add tabulate as a dependency